File: database/mssql.php
<?php
$host = "centriumres.com.localdomain.ee";
$dbSettings = [
    "PWD" => "D3v", // see below for live
    "Database" => "Serenity",
    "UID" => "Dev", // see below for live
    "CharacterSet" => "UTF-8", // otherwise, characters in Property details truncates return from SQL driver
];


$userName = "Dev";
$password = "D3v";
$database = "Serenity";


// $pdo = new PDO("sqlsrv:Server=$host;Database=$database", $userName, $password);
// return $pdo;

  $conn = sqlsrv_connect($host, $dbSettings);

  $sql = "select * from XMLBookingLogin WHERE Login ='sdfasfasdfsa'";
$rtn=array();
$stmt = sqlsrv_prepare()
  $stmt = sqlsrv_query($conn, $sql);
  while($row = sqlsrv_fetch_array($stmt, SQLSRV_FETCH_ASSOC)) {
    $rtn[]=$row;
  }
  
  var_dump($rtn);

File: python/database/mssql.py
import pyodbc
conn = pyodbc.connect('DRIVER={ODBC Driver 17 for SQL Server};'
#conn = pyodbc.connect('DRIVER={/opt/microsoft/msodbcsql17/lib64/libmsodbcsql-17.4.so.2.1};'
                      'Server=192.168.10.171;'
                      'UID=Dev;'
                      'PWD=D3v;'
                      'Database=Serenity;'
                      'Trusted_connection=yes;')

cursor = conn.cursor()
cursor.execute('select * from booking order by BookingID desc')

for row in cursor:
  print(row)
